gui-wm/hyprland/hyprland-9999.ebuild

```diff
@@ -63,6 +63,20 @@ BDEPEND="
 
 PATCHES=( "${FILESDIR}/${PN}-0.10.3beta-system-wlroots.patch" )
 
+src_prepare() {
+	default
+
+	# Nvidia patch
+	if use video_cards_nvidia; then
+		sed -i "s|glFlush();|glFinish();|" \
+			"${S}/subprojects/wlroots/render/gles2/renderer.c" || die "Nvidia patch failed"
+
+		eapply "${FILESDIR}/0001-Screencast-for-Nvidia.patch"
```

Comment:
Patch file is not included

Reply:
Sorry I forgot it. I'll add. The patch below can be used:

```
types/output/render.c | 17 +----------------
 1 file changed, 1 insertion(+), 16 deletions(-)

diff --git a/subprojects/wlroots/types/output/render.c b/subprojects/wlroots/types/output/render.c
index 8e0895a6..c271f72a 100644
--- a/subprojects/wlroots/types/output/render.c
+++ b/subprojects/wlroots/types/output/render.c
@@ -338,22 +338,7 @@ struct wlr_drm_format *output_pick_format(struct wlr_output *output,
 }
 
 uint32_t wlr_output_preferred_read_format(struct wlr_output *output) {
-	struct wlr_renderer *renderer = output->renderer;
-	assert(renderer != NULL);
-
-	if (!renderer->impl->preferred_read_format || !renderer->impl->read_pixels) {
-		return DRM_FORMAT_INVALID;
-	}
-
-	if (!output_attach_back_buffer(output, &output->pending, NULL)) {
-		return false;
-	}
-
-	uint32_t fmt = renderer->impl->preferred_read_format(renderer);
-
-	output_clear_back_buffer(output);
-
-	return fmt;
+	return DRM_FORMAT_XRGB8888;
 }
 
 bool output_is_direct_scanout(struct wlr_output *output,
-- 
2.39.2
```

Reply:
Please add it in the PR.